packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx

```diff
@@ -74,8 +74,8 @@ const Page = (): React.ReactElement => {
           'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?type=design&node-id=16767-706689&t=6FVL3Ha33gwM45Cm-0',
       }}
     >
-      <Title>Usage</Title>
-      <Sandbox editorHeight={600}>
+      <Heading size="large">Usage</Heading>
```

Comment:
I think the codemod will also need to fall back to `size="large"` if the size isn't explicitly defined.